recipes-support/provisioning/esec-provisioning.bb

```diff
@@ -0,0 +1,18 @@
+SUMMARY = "Provisioning tools for ESEC IoT Device Manager plattform"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
```

Comment:
This is incorrect. The script is licensed under GPLv2, not MIT. You have to either license the script under MIT or set the correct license type here.

Reply:
changed to GPL-2.0